pyirf/binning.py

```diff
@@ -152,6 +153,9 @@ def calculate_bin_indices(data, bins):
         Indices of the histogram bin the values in data belong to.
         Under- and overflown values will have values of `UNDERFLOW_INDEX`
         and `OVERFLOW_INDEX` respectively.
+
+    valid: np.ndarray[bool]
+        Boolean mask indicated if a given value fell into the over- or underflow bins.
```

Comment:
I'd rephrase slightly, as `valid` indicates those values that are neither under- nor overflow. Maybe

Boolean mask indicating those values as valid that did not fell into the over- or underflow bins.